apps/emqx/include/logger.hrl

```diff
@@ -119,7 +119,13 @@ end).
 -endif.
 
 %% print to 'user' group leader
--define(ULOG(Fmt, Args), io:format(user, Fmt, Args)).
--define(ELOG(Fmt, Args), io:format(standard_error, Fmt, Args)).
+-define(ULOG(Fmt, Args),
```

Comment:
This is probably a change which deserves a proper change log, and release in 5.8